modules/claims_api/spec/requests/v2/power_of_attorney_requests/decisions/create/rswag_spec.rb

```diff
@@ -94,13 +117,22 @@
         end
 
         before do |example|
+          allow(ClaimsApi::PowerOfAttorneyRequest).to(
+            receive(:find_by).and_return(nil)
+          )
           mock_ccg(scopes) do
             submit_request(example.metadata)
           end
         end
 
         after do |example|
-          example.metadata[:response][:content] ||= { 'application/json' => { examples: {} } }
+          example.metadata[:response][:content] ||= { 'application/json' => { examples:
+          { errors: [{
+            status: '404',
+            code: '108',
+            title: 'Resource not found',
+            detail: 'Could not find Power of Attorney request with id: 348fa995-5b29-4819-91af-13f1bb3c7d77'
+          }] } } }
```

Comment:
Interesting that we need this block. Does the following not work:

```suggestion
          example.metadata[:response][:content] = { 
            'application/json' => { 
              example: JSON.parse(response.body, symbolize_names: true)
           }
         }
```
?